app/packages/components/src/components/JSONPanel/panel.module.css

```diff
@@ -17,6 +17,57 @@
   pointer-events: all;
 }
 
+.lookerPanelV2 {
```

Comment:
I'd propose we remove all old/unused json viewer code. This class is nearly identical to the one above, no need for both probably